src/main/java/seedu/address/logic/parser/AddressBookParser.java

```diff
@@ -153,6 +162,9 @@ public Command parseCommand(String userInput) throws ParseException {
             case INTRO:
                 return new IntroCommand();
 
+            case SHOW_REMINDER:
+                return new ShowReminderEventCommand();
+
```

Comment:
This is separated from the other 2 reminder commands when they should be grouped under another command type for reminder. Instead of belonging to the default branch.